sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md

```diff
@@ -0,0 +1,212 @@
+| Approver   | _#! TO-DO_         |    |          |
+| Authorizer | _#! TO-DO_         |    |          |
+
+### 4. Purpose
+To access controlled access data within the European Genomic Data Infrastructure a research user must apply for authorisation to access the data from the 1+ Million Genomes Data Access Committee (1+MG DAC). The 1+MG DAC (a.k.a. 1+MG EDIC) will **review applications** to ensure they conform to the 1+MG Data Governance requirements, and **make recommendations** to the relevant National Coordination Points (NCPs) who may agree or disagree with the recommendation.
+
+### 5. Scope
+This SOP covers the process **from when a data access application is received by the 1+MG DAC to when the application is approved or rejected** by the 1+MG DAC. The **input** is a completed application form, and the **output** is a notification of approval or rejection of the request for authorisation to access the controlled access data described in the application.
+
+### 6. Introduction and Background Information
+The 1+MG DAC is responsible for recommending the approval or rejection of requests for access to data held within the GDI to the relevant NCP(s). Additionally, the 1+MG DAC will follow a review process when the NCP disagrees with its recommendation (see [GDI-SOP0002_NCPs-veto-EDIC-decision.md](../node-specific/GDI-SOP0002_NCPs-veto-EDIC-decision.md)). Once an agreement is reached, or the review process completes, the data user will be informed of the decision.
+
+### 7. Summary or Context Diagram
+The following diagram summarises the step-by-step process for the recommendation by the 1+MG DAC of approval or rejection of data access applications.
+````mermaid
+graph TD
+    A[**Start**: Data Access<br>Application Received]
+    B1{**Step 1**:<br> Application Checked<br> by 1+MG DAC}
+    B2[**Step 2**: Application<br>Recommended for Approval]
+    B2.2{Response<br>from NCPs?}
+    B3[**Step 3**: No NCP Response<br> - Approval Confirmed]
+    B4[**Step 4**: NCP Agrees<br> - Approval Confirmed]
+    B5[**Step 5**: NCP Disagrees<br> - NCP Review<br> Process Initiated]
+    B6[**Step 6**: NCP Review<br> Upholds Recommendation<br> - Approval Confirmed]
+    B7[**Step 7**: NCP Review<br> Overrules Recommendation<br> - Application Rejected]
+    B8[**Step 8**: Review Delayed<br> - Inform Data User]
+    B9[**Step 9**: Approval confirmed<br> - Inform Data User]
+    B10[**Step 10**: Rejection confirmed<br> - Inform Data User]
+
+    A --> B1
+    B1 -->|Application<br>is approved| B2
+    B2 --> B2.2
+    B2.2 -->|No NCP Response| B3
+    B2.2 -->|NCP Agrees| B4
+    B2.2 -->|NCP Disagrees| B5
+    B3 --> B9
+    B4 --> B9
+    B6 --> B9
+    B5 -->|Recommendation Upheld| B6
+    B5 -->|Recommendation Overruled| B7
+    B5 -->|Review Delayed| B8
+    B1 -->|Application<br>is rejected| B10
+    B7 --> B10
+    B8 -->|Follow up<br> review process| B5
+````
+
+### 8. Procedure
+#### 1. Initiate application check
```

Comment:
I think there were comments about consistent & clear / unambiguous numbering previously in a different SOP (the ISM one?). both ### headers & #### headers are both numbered with 1., 2., 3. etc. I think we need to come up with a system that we apply across all SOPs, if possible. Here, what about using (1), (2), (3) etc for the #### headers??